style/options_basic.css

```diff
@@ -381,4 +369,39 @@ body.rtl #yarpp-optin-learnmore {
 	max-height: 500px;
 	width: 100%;
 	overflow: auto;
-}
\ No newline at end of file
+}
```

Comment:
removed these styles from yarpp pro css, so put them into the basic css file instead.